Tasks/AzureRmWebAppDeployment/AzureRMUtil.ts

```diff
@@ -124,7 +124,15 @@ function getUpdateHistoryRequest(webAppPublishKuduUrl: string, isDeploymentSucce
 
     var collectionUrl = tl.getVariable('system.TeamFoundationCollectionUri'); 
     var teamProject = tl.getVariable('system.teamProject');
-
+	
+	var type = "Deployment";
+	var commitId = tl.getVariable('build.sourceVersion');
+	var repoName = tl.getVariable('build.repository.name');
+	var repoProvider = tl.getVariable('build.repository.provider');
+	var slotName = tl.getInput('SlotName');
+	if(slotName==null)
```

Comment:
General rule, do this wherever applicable. Add space before and after any operator,  here it is ( == )